src/components/loader-spinner/loader-spinner.test.tsx

```diff
@@ -302,9 +326,10 @@ describe("when custom props are passed", () => {
 
   it.each([1, 2, 3, 5, 10])(
     "when 'animationTime' is passed as `%s` the animation duration is the correct time",
-    (animationTimes) => {
+    async (animationTimes) => {
```

Comment:
comment: I could be wrong on this but I don't think we need the tests to be async as we're mocking the hook etc. It's not critical but it will make the test suite run slower as it `awaits` so if we don't need them to be async I think it's better to not have them be etc